modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional-enabled-capabilities{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the jq package.
+
+.Procedure
+
+. Add to `additionalEnabledCapabilities` or adjust `baselineCapabilitySet` by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type merge -p '{"spec":{"capabilities":{"baselineCapabilitySet":"vCurrent"}}}'
+----
+
++
+.Example output
+[source,terminal]
+----
+["openshift-samples"]
+----
+
+. Append to marketplace by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion/version --type merge -p '{"spec":{"capabilities":{"additionalEnabledCapabilities":["openshift-samples", "marketplace"]}}}'
+----
+
++
+[IMPORTANT]
+====
+It is not possible to disable a capability which is already enabled in a cluster. The cluster version Operator (CVO) continues to reconcile the capability which is already enabled in the cluster.
+====
+
+
+If you try to disable a capability, then CVO shows the divergent spec:
+[source,terminal]
+----
+oc get -o json clusterversion version | jq -r '.status.conditions[] | select(.type == "ImplicitlyEnabledCapabilities") | .status + " " + .reason + ": " + .message'
+----
+
+.Example output
+[source,terminal]
+----
+True CapabilitiesImplicitlyEnabled: The following capabilities could not be disabled: openshift-samples
+----
+
+[NOTE]
+====
+During cluster upgrades some new capability might get enabled, if the corresponding manifest for the capability is now part of a capability, which was enabled in the cluster before the upgrade is performed.
```

Comment:
This is a confusing concept so I think we need to be sure we're clearly explaining it. Something like:
> During cluster updates it's possible that a given capability could be implicitly enabled. For example, if a resource was already running on the cluster, and that resource's manifest has changed as part of the update to contain a capability, that capability will always be enabled.

Reply:
I will reword @jottofar 's suggestion as below. @jottofar let me know if this does not sound right to you?
```
During cluster updates it's possible that a new capability could get implicitly enabled. For example, if an existing capability ( i.e. a capability which is enabled before the upgrade) has changed to include manifests from another capability which was not enabled before the upgrade. 
```

Reply:
The way I worded it is the more general case and more accurate. It doesn't matter whether the manifest was part of an enabled capability before the upgrade. It only matters that it was deployed on the cluster (applied) before the upgrade. It may not have been part of a capability before the upgrade - doesn't matter. Now the upgrade occurs and if the manifest that represents that resource now has any capabilities they must be enabled because the resource can't be "disabled" (deleted).

Reply:
How about:
> During cluster upgrades it's possible that a given capability could be implicitly enabled. If a resource was already running on the cluster before the upgrade any capabilities that the resource is a part of will be enabled. For example, assume as part of the upgrade a resource that is already running on the cluster has been changed to be part of the `marketplace` capability. The `marketplace` capability will be enabled. If an administrator has not explicitly enabled the `marketplace` capability it will be `implicitly` enabled by the system.

Reply:
@jottofar Your first suggestion was easy to understand. However I'm okay with the above suggestion. Please let me know which one I should include.

Reply:
I like the last one https://github.com/openshift/openshift-docs/pull/47915#discussion_r925722231 . so lets go ahead with it.